content/contact.md

```diff
@@ -5,15 +5,18 @@ title = "Contact"
 The best place to get information and connect is on Matrix itself - starting in
 [Matrix HQ (#matrix:matrix.org)](https://matrix.to/#/#matrix:matrix.org) or in the [Office of the Matrix.org Foundation (#foundation-office:matrix.org)](https://matrix.to/#/#foundation-office:matrix.org).
```

Comment:
I wonder if this is still true. In my experience Matrix HQ is a large room with little interesting activity, and I'm not sure most people working on Matrix hang in there.

This is also a historical room with a lot of state that can melt down most self-hosted deployment. I'm not convinced this is a good first experience.

Not blocking the PR on this but we should consider removing it / routing people elsewhere.